src/audiovideofacade/DefaultAudioVideoFacade.ts

```diff
@@ -394,6 +395,12 @@ export default class DefaultAudioVideoFacade implements AudioVideoFacade {
     });
   }
 
+  getVideoInputQualitySettings(): VideoQualitySettings {
```

Comment:
Do we want to specify `| null` in the return type as in other facade APIs? We haven't enabled the null check in TS config though.